extensions/2.0/Vendor/FB_ngon_encoding/schema/glTF.FB_ngon_encoding.schema.json

```
@@ -0,0 +1,15 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "FB_ngon_encoding glTF extension",
```

Update schema with new glTF schema version, add ID, and adjust title. The file should be renamed to `mesh.primitive.FB_ngon_encoding.schema.json` to clarify that this extends glTF mesh primitives.

```suggestion
{
    "$schema": "https://json-schema.org/draft/2020-12/schema",
    "$id": "mesh.primitive.FB_ngon_encoding.schema.json",
    "title": "FB_ngon_encoding glTF Mesh Primitive Extension",
```

---

@aaronfranke I recommend taking this under omi or godot namespaces and doing the updates.

---

See https://github.com/saturday06/VRM-Addon-for-Blender/issues/101